iron-overlay-manager.html

```diff
@@ -67,55 +64,20 @@
 
     /**
      * The deepest active element.
-     * @type {Element|undefined} activeElement the active element
+     * @type {Element} activeElement the active element
      */
     get deepActiveElement() {
```

Comment:
_No action required_, but we need to make this a global Polymer utility or something.